GAPIC-generated system tests broken on Kokoro

From: https://source.cloud.google.com/results/invocations/3b9800e0-277d-4e82-bf21-bc7b72d2eef3/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Ferror_reporting/log

```python

=================================== FAILURES ===================================
____________ TestSystemReportErrorsService.test_report_error_event _____________

self = <test_system_report_errors_service_v1beta1.TestSystemReportErrorsService object at 0x7f7e164d7cf8>

    def test_report_error_event(self):
>       project_id = os.environ['PROJECT_ID']

tests/system/gapic/v1beta1/test_system_report_errors_service_v1beta1.py:25:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = environ({'SYSTEMROOT': '', 'KOKORO_GFILE_DIR': '/tmpfs/src/gfile', 'HOSTNAME': 'kokoro-gcp-ubuntu-prod-1154164688', 'K...eta1/test_system_report_errors_service_v1beta1.py::TestSystemReportErrorsService::()::test_report_error_event (call)'})
key = 'PROJECT_ID'

    def __getitem__(self, key):
        try:
            value = self._data[self.encodekey(key)]
        except KeyError:
            # raise KeyError with the original key value
>           raise KeyError(key) from None
E           KeyError: 'PROJECT_ID'
```

The following codegen-derived system tests files use `os.environ['PROJECT_ID']`:

- `bigquery_datatransfer/tests/system/gapic/v1/test_system_data_transfer_service_v1.py`
- `container/tests/system/gapic/v1/test_system_cluster_manager_v1.py`
- `dataproc/tests/system/gapic/v1/test_system_cluster_controller_v1.py`
- `dataproc/tests/system/gapic/v1beta2/test_system_cluster_controller_v1beta2.py`
- `error_reporting/tests/system/gapic/v1beta1/test_system_report_errors_service_v1beta1.py`
- `iot/tests/system/gapic/v1/test_system_device_manager_v1.py`
- `monitoring/tests/system/gapic/v3/test_system_metric_service_v3.py`
- `pubsub/tests/system/gapic/v1/test_system_publisher_v1.py`
- `trace/tests/system/gapic/v1/test_system_trace_service_v1.py`
- `trace/tests/system/gapic/v2/test_system_trace_service_v2.py`

The "standard" environment variable for such purposes is `GOOGLE_CLOUD_PROJECT`. It looks to me like we used to have `PROJECT_ID` defined as a [CircleCI environment variable](https://circleci.com/gh/googleapis/google-cloud-python/edit#env-vars):  we should likely add it for Kokoro too.

--- a/.kokoro/build.sh
+++ b/.kokoro/build.sh
@@ -30,6 +30,9 @@ export FIRESTORE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/firebase-credential
 # Setup service account credentials.
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
 
+# Setup project id.
+export PROJECT_ID=$(cat "${KOKORO_GFILE_DIR}/project-id.json")
+
 # Find out if this package was modified.
 # Temporarily use Thea's fork of ci-diff-helper w/ Kokoro support.
 python3.6 -m pip install --quiet git+https://github.com/theacodes/ci-diff-helper.git

